flow/connectors/postgres/cdc.go

```diff
@@ -504,7 +513,7 @@ func (p *PostgresCDCSource) processMessage(batch *model.CDCRecordStream, xld pgl
 		if p.relationMessageMapping[msg.RelationID] == nil {
 			p.relationMessageMapping[msg.RelationID] = convertRelationMessageToProto(msg)
 		} else {
-			return p.processRelationMessage(xld.WALStart, convertRelationMessageToProto(msg))
+			return p.processRelationMessage(currentClientXlogPos, convertRelationMessageToProto(msg), flowJobName)
```

Comment:
why did this change from `xld.WALStart` to `currentClientXlogPos` ?

Reply:
`xld.WALStart` is 0 for `RelationMessages`, so using this instead.